recipes/cftime/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "1.0.0a2" %}
+
+package:
+  name: cftime
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/cftime/cftime-{{ version }}.tar.gz
```

Comment:
This is not on PyPI yet, right?

Reply:
Correct. I'll try to push it this evening.